contracts/CapabilityDelegator.cdc

```diff
@@ -7,56 +7,56 @@
 /// private `Delegator` can only be borrowed from the child account when you have access to the full `ChildAccount` 
 /// resource.
 ///
-pub contract CapabilityDelegator {
+access(all) contract CapabilityDelegator {
 
     /* --- Canonical Paths --- */
     //
-    pub let StoragePath: StoragePath
-    pub let PrivatePath: PrivatePath
-    pub let PublicPath: PublicPath
+    access(all) let StoragePath: StoragePath
+    access(all) let PrivatePath: PrivatePath
```

Comment:
should this not go away?

Reply:
Yes, all PrivatePath types will need to be deleted